distribution.pm

```diff
@@ -112,25 +112,26 @@ sub script_run ($self, $cmd, @args) {
         {
             timeout => $bmwqemu::default_timeout,
             output => '',
-            quiet => undef
+            quiet => undef,
+            max_interval => testapi::DEFAULT_MAX_INTERVAL
```

Comment:
Would be good to also document the new option

Reply:
[Doc PR](https://github.com/os-autoinst/os-autoinst/pull/2752)